app/src/main/java/ca/marshallasch/veil/MainActivity.java

```diff
@@ -42,8 +43,14 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+
        // getFragmentManager().beginTransaction().add(R.id.fragment_container,  /* Put the frag here*/).commit();
 
+        FragmentTransaction transaction = getFragmentManager().beginTransaction();
+        transaction.addToBackStack(null);
```

Comment:
Do not add the initial fragment to the back stack, otherwise when you press the back button you will have an empty page that you will be stuck on.

Reply:
resolved